docs/UserGuide.md

```diff
@@ -310,59 +324,96 @@ Example command usages from the current student list shown in Figure 1:
 * `find_student Lee` returns "John Lee" and "Samuel Lee"
 * `find_student Johnz Lee` returns "John Lee", "Johnz Tan" and "Samuel Lee"
 
+Example 2:
+1. You want to find students whose name contains alex, david and roy.
+
+2. Type `find_student alex david roy` and press <kbd>enter</kbd>.
+
+![find_student_example](images/enhao/find_student%20-%20edited.png)
+
+3. The student and session list will be filtered and only contains students and sessions of students whose name contains the keywords specified.
+
+![find_student_result_example](images/enhao/find_student_result%20-%20edited.png)
+
+
 ##### Editing a student: `edit_student`
 
 Edits an existing student's detail(s) in TutorBuddy.
 
 Format: `edit_student STUDENT_INDEX [n/NAME] [p/STUDENT_PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [l/STUDY_LEVEL] [g/GUARDIAN_PHONE_NUMBER] [r/RELATIONSHIP_WITH_GUARDIAN]` <br>
 * `STUDENT_PHONE_NUMBER` and `GUARDIAN_PHONE_NUMBER` should be in Singapore's phone formatting (i.e. starting with either 6, 8 or 9 with 8 digits)
 
-Examples:
-* `edit_student 1 p/98825432 e/alexy@example.com a/Alex street, Block 123, #01-01` edits the student at index 1's `STUDENT_PHONE_NUMBER`, `EMAIL`, and `ADDRESS`
+Example:
+1. You would like to edit **Alex Yeoh** student's phone number, email address and address as they are wrong. 
```

Comment:
maybe 'edit Alex Yeoh's phone number' would be enough? so omit the '**student**'. Or we can say 'Student Alex Yeoh' instead.

Reply:
Got it! Updated in the latest commit.